app/assets/javascripts/application.js

```diff
@@ -69,12 +71,71 @@ $(function(){
     /* This is the same for all */
   });
 
+  // group show page side bar
+  $('#side-task li').hover(function(){
+    /* This is the same for all */
+    $('.pop-up').removeClass('hide');
+    var popUpOffSet = $(this).offset();
+    popUpOffSet.left += $(this).width();
+    $('.pop-up').offset(popUpOffSet);
+    var popUpData = JSON.parse($(this).attr('meta-data'));
+    /* This is the same for all */
+
+    /* This is the different base on json object for all */
+    var wrapperDiv = document.createElement('div');
+    var titleDiv = document.createElement('div');
+    titleDiv.innerHTML = popUpData.title;
+    var descriptionDiv = document.createElement('div');
+    descriptionDiv.innerHTML = popUpData.description;
+    var completedDiv = document.createElement('div');
+    completedDiv.innerHTML = 'completed? ' + popUpData.completed;
+    wrapperDiv.appendChild(titleDiv);
+    wrapperDiv.appendChild(descriptionDiv);
+    wrapperDiv.appendChild(completedDiv);
+    /* This is the different base on json object for all */
+
+
+    /* This is the same for all */
+    $('#pop-up-content').html(wrapperDiv);
+    /* This is the same for all */
+  });
+
+  // group show page side bar
+  $('#side-contact li').hover(function(){
+    /* This is the same for all */
+    $('.pop-up').removeClass('hide');
+    var popUpOffSet = $(this).offset();
+    popUpOffSet.left += $(this).width();
+    $('.pop-up').offset(popUpOffSet);
+    var popUpData = JSON.parse($(this).attr('meta-data'));
+    /* This is the same for all */
+
+    /* This is the different base on json object for all */
+    var wrapperDiv = document.createElement('div');
+    var companyDiv = document.createElement('div');
+    companyDiv.innerHTML = popUpData.company_name;
+    var nameDiv = document.createElement('div');
+    nameDiv.innerHTML = popUpData.first_name + ' ' + popUpData.last_name;
```

Comment:
Identifier 'first_name' is not in camel case.<br>Identifier 'last_name' is not in camel case.